entrypoint.sh

```diff
@@ -24,15 +24,29 @@ if [[ ! "$1" =~ ^(fmt|init|plan|validate)$ ]]; then
   exit 1
 fi
 
+if [ "$2" ]; then
+  input_raw="$2"
+fi
+
 comment="$4"
+file="$5"
+
+if [ "$file" ]; then
```

Comment:
```suggestion
if [ -s "$file" ]; then
```

test if its a non-empty file

Reply:
That is the default for test:

```
man test
[...]
string        True if string is not the null string.
```

Passing a file that does not exist should result in an error and not silently lead to `input_file` being used if it was passed. We are not strict enough for that case at the moment.